README.md

```diff
@@ -7,8 +7,8 @@ Query API for the Tezos blockchain
 ## Running Conseil
 
 Conseil has two entry points:
-- `src/main/tech/cryptonomic/Conseil/Conseil.scala` runs a server exposing a RESTful API.
-- `src/main/tech/cryptonomic/Conseil/Lorre.scala` is a background process that runs on loop and writes Tezos blockchain data to database.
+- `src/main/tech/cryptonomic/Conseil/Conseil.scala` runs a server exposing a RESTful API from locally stored blockchain data.
```

Comment:
What does "locally sourced" mean?

Reply:
I meant to say that it uses a separate db over reading the data from the chain itself...
I guess I might rephrase to be more correct...